src/Glpi/Controller/Install/Step7Controller.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Controller\Install;
+
+use Glpi\Http\Firewall;
+use Glpi\Security\Attribute\SecurityStrategy;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Toolbox;
+use Glpi\Controller\AbstractController;
+use Glpi\Progress\ProgressChecker;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+use Symfony\Component\Routing\Attribute\Route;
+
+class Step7Controller extends AbstractController
```

Comment:
I am not sure that using a step number in the name is a good idea. We will probably refactor the install process in a later version, and this step may then be the fifth step, or the ninth step, and we would probably not want rename the controller and the URLs just for that.

By the way, maybe we can just have a single `InstallController` instead of splitting it into one controller for each step. Each step code is pretty light (and will be even lighter in the future as we will probably move most of its logic into a dedicated service that would be also used by the install CLI command).

Reply:
Let's rename it `InstallController` then 👍